dcos/http.py

```diff
@@ -414,7 +428,7 @@ def _get_dcos_acs_auth(username, password, hostname):
 
         # using private method here, so we don't retry on this request
         # error here will be bubbled up to _request_with_auth
-        response = _request('post', url, json=creds)
+        response = _request('post', url, json=creds, verify=_verify_ssl())
```

Comment:
I'll move the call to `silence_requests_warning` outside of `_verify_ssl` so there aren't side effects.